### DISCUSSION TITLE
Academic Verification (AZURE)

### DISCUSSION THREAD

**Discussion by LiMaKen** [community/community#129632] [GitHub Education]
### Hi everyone,

Hello 
I have a student development package and I'm authenticating with Azure using my Github account but I'm facing this problem. 
What should I do now?
Has anyone here encountered such a situation?

I would be happy if someone could answer me.
![image](https://github.com/community/community/assets/86464924/0f0ae509-6ea2-446c-b1f0-ba54dfd0c739)

**Comment by yusufekorman** [comment]
same thing

**Comment by cristiandpto23** [comment]
same problem
